server.js

```diff
@@ -0,0 +1,80 @@
+import Fastify from 'fastify'
+import { TwitterApi } from 'twitter-api-v2'
+
+export const ERR_MISSING_TWITTER_KEYS = 'Missing required Twitter access keys!'
+export const ERR_MISSING_INPUTS =
+  'Missing required inputs: action, release or repository!'
+export const ERR_TWITTER =
+  'Sending message failed due to an error from Twitter: '
+export const MSG_REPO_IS_PRIVATE =
+  'Repository is private, no message is sent to Twitter.'
+export const MSG_REPO_IS_NOT_PUBLISHED =
+  'Repository is not published so no message is sent to Twitter.'
+export const MSG_MESSAGE_LOG = 'Sending message: '
+export default function buildServer(config) {
+  const REPOSITORY_PUBLISHED = 'published'
```

Comment:
this constant doesn't need to live inside this function. also, it's misnamed. it's not the repo that get's published, it's a release that is published in the repo